src/utils/stats.js

```diff
@@ -0,0 +1,32 @@
+const ReactGA = require('react-ga')
+
+const GA_TRACKING_ID = 'UA-74131346-14'
+const GA_ADDRESS = 'https://www.prisma.io/gastats.js'
+const COLLECT_ADDRESS = 'https://stats.prisma.workers.dev'
+
+module.exports = {
+  init() {
+    ReactGA.initialize(GA_TRACKING_ID, {
+      gaAddress: GA_ADDRESS,
+    })
+
+    const ga = ReactGA.ga()
+    ga(u => {
+      // Override sendHitTask to proxy tracking requests
+      u.set('sendHitTask', model => {
+        const xhr = new XMLHttpRequest()
+        xhr.open('POST', COLLECT_ADDRESS, true)
+        xhr.send(model.get('hitPayload'))
+      })
+    })
+  },
+
+  trackPage(page) {
+    const { hostname } = window.location
+
+    if (hostname.includes('netlify') || hostname.includes('localhost')) {
```

Comment:
```suggestion
    const { host } = window.location

    if (host.includes('netlify') || host.includes('localhost')) {
```

Tiny change when I remembered that hostname is not as well supported (not supported by Opera 🤷‍♂️) as host (it's almost the same thing)
<img width="825" alt="Screen Shot 2020-05-18 at 10 45 10" src="https://user-images.githubusercontent.com/1328733/82200014-c3a97800-98fe-11ea-98a7-aa0c77f98123.png">
	https://developer.mozilla.org/en-US/docs/Web/API/Location/host